items.csv

```diff
@@ -1,20 +1,1222 @@
+acacia_boat,1,acacia_planks,5
+acacia_button,1
+acacia_door,3,acacia_planks,6
+acacia_fence,3,acacia_planks,4,stick,2
+acacia_fence_gate,1,stick,4,acacia_planks,2
+acacia_planks,4
+acacia_pressure_plate,1,acacia_planks,2
+acacia_sign,3,acacia_planks,6,stick,1
+acacia_slab,6,acacia_planks,3
+acacia_stairs,4,acacia_planks,6
+acacia_trapdoor,2,acacia_planks,6
+acacia_wood,3,acacia_log,4
 activator_rail,6,iron_ingot,6,redstone_torch,1,stick,2
+activator_rail,6,iron_ingot,6,stick,2,redstone_torch,1
+andesite,2
```

Comment:
There are a few recipes with duplicate entries (and some with even more than two copies).

Reply:
Some duplicates are intended, some are not. The recipes with less ingredients were improperly scraped. Other recipes that are entirely missing ingredients used some sort of unusual naming system that Power query couldnt parse. I've been manually fixing those (for example all the wool recipes.)

Reply:
Why are any duplicates intended?

Reply:
Some items have multiple ways to acquire them. Take for example sugar it can be acquired from honey bottles or sugar cane. Dye can also be made from multiple flowers.